### PR TITLE
Add unsupported LINQ types info

### PR DESCRIPTION
**Pull request by MongoCaleb**
### Jira

https://jira.mongodb.org/browse/DOCSP-14921

### Staged Changes
https://docs-mongodbcom-staging.corp.mongodb.com/realm/docsworker-xlarge/DOCSP-14921/sdk/dotnet/crud/filter/#unsupported-linq-operators

### CODE REVIEW

File: source/sdk/dotnet/crud/filter.txt [hunk 1]
```diff
@@ -224,6 +231,50 @@ table. Regex-like wildcards allow more flexibility in search.
...
+
+.. list-table::
+   :header-rows: 1
+   :widths: 40 60
+
+   * - Category
+     - Unsupported Operators
+
+   * - | Grouping
+     -   - ``Group``
```
Comment (cbullinger) [inline source/sdk/dotnet/crud/filter.txt:248]: Should this be `GroupBy`?

File: source/sdk/dotnet/crud/filter.txt [hunk 2]
```diff
@@ -63,6 +63,13 @@ An expression consists of one of the following:
 - An operator
 - A value of any type used by realm (string, date, number, boolean, etc.)
 
+.. note:: 
+
+   Currently, the Realm .NET SDK doesn't support all of the LINQ operators. Refer
```
Comment (cbullinger) [inline source/sdk/dotnet/crud/filter.txt:68]: I'd suggest spelling out as `does not` to emphasize the negative

Comment (cbullinger) [inline source/sdk/dotnet/crud/filter.txt:68]:
```suggestion
   Currently, the Realm .NET SDK does not support all of the LINQ operators. Refer 
```

File: source/sdk/dotnet/crud/filter.txt [hunk 3]
```diff
@@ -224,6 +231,50 @@ table. Regex-like wildcards allow more flexibility in search.
...
+           is supported as long as you select the Realm object itself and not a 
+           derivative:
+           
+           ``var tasks = from t in realm.All<Task>() where t.Assignee == "Caleb" select t;``
+
+   * - | Concatenation
+     -   - ``Concat``
+         - ``Join``. While ``Join`` is not supported, it is not needed with Realm. 
+           Instead of using keys in a Join statement, as you would in a traditional 
+           relational database, instead reference another type as a property.
```
Comment (cbullinger) [inline source/sdk/dotnet/crud/filter.txt:274]:
```suggestion
           relational database, you can reference another type as a property.  
```

File: source/sdk/dotnet/crud/filter.txt [hunk 4]
```diff
@@ -224,6 +231,50 @@ table. Regex-like wildcards allow more flexibility in search.
...
+   :header-rows: 1
+   :widths: 40 60
+
+   * - Category
+     - Unsupported Operators
+
+   * - | Grouping
+     -   - ``Group``
+
+   * - | Sets
```
Comment (cbullinger) [inline source/sdk/dotnet/crud/filter.txt:250]: The seemingly random order stands out especially in this group -- is there a reason the SDK docs put them in this order or could we possibly put them in alpha order?

**Review by cbullinger — APPROVED** [review]
A couple suggestions, but otherwise LGTM. Good thinking on the table format.

File: source/sdk/dotnet/crud/filter.txt (hunk 1 above)
Comment (MongoCaleb) [reply source/sdk/dotnet/crud/filter.txt:248]: Great catch!

**Pull request merged**